nextcloud/server
# Fix icons from settings page disappears when nav item overflows

Fixing #23849 bug. 
Issue was general for the settings page: when a navigation item text overflows, svg icons disapeared. 
Some minor changes in the apps.scss file fixed the issue. 

**How to test:**

1. Go on the settings page
2. Inspect the code and add text in the navigation item
3. See the icon stays and the overflow is well handled
